app/plugins/prometheus.md

```diff
@@ -0,0 +1,183 @@
+---
+id: page-plugin
+title: Plugins - Prometheus
+header_title: Prometheus
+header_icon: https://konghq.com/wp-content/uploads/2018/06/prometheus.png
+breadcrumbs:
+  Plugins: /plugins
+nav:
+  - label: Usage
+    items:
+      - label: Log Format
+description: |
+    Expose metrics related to Kong and the upstream services in [Prometheus](https://prometheus.io/docs/introduction/overview/) Exposition format which can be scrapped by a Prometheus Server.
+
+params:
+  name: prometheus
+  api_id: true
+  service_id: true
+  route_id: true
+
+---
+
+### Reading metrics into Prometheus
+
+Metrics are availble on the Admin API at `http://localhost:8001/metrics`
+endpoint. Note the URL to the Admin API will be specific to your installation.
```

Comment:
It's important to mention the following bit here:
```
This plugin records and exposes metrics at node-level.
Prometheus server will need to discover all Kong nodes via your service discovery mechanism, scrape each of the Kong nodes.
```

Reply:
How would this work for a node that is proxy only?  It sounds like a proxy only node would not have the admin /metrics available at all.

Reply:
Right, so in that case, one can either use a custom NGINX template or use NGINX directive injection mechanism (shipped with 0.14) to expose these metrics on a separate port.

Reply:
Howsabout suggesting that in the doc? Seems like a helpful tip!

Reply:
@hbagdi @coopr  I cleaned up the explanation and hopefully make it easier to understand for someone reading it for the first time.